src/api/product/controllers/item.ts

```diff
@@ -8,19 +8,21 @@ import { processProductItem } from "../../../functions/content-types/product/ite
 export default factories.createCoreController(
   "api::product.item",
 
-  ({ strapi }) => ({
+  () => ({
```

Comment:
question(non-blocking): It might be useful for folks looking at this file in the future to know they have access to the `strapi` parameter. Is there an easy way to keep it?